code/game/objects/structures/stool_bed_chair_nest/xeno_nest.dm

```diff
@@ -12,7 +12,7 @@
 	var/on_fire = 0
 	var/resisting = 0
 	var/resisting_ready = 0
-	var/nest_resist_time = 1200
+	var/nest_resist_time = 800 //1.5 mins
```

Comment:
We have a define for this btw `1.5 MINUTES` works because minutes is `#define MINUTES *600`